test/bf-projects/bf_simple_v10/project_setup.py

```diff
@@ -3,4 +3,4 @@
 PROJECT_NAME = "bf_simple_v10"
```

Comment:
maybe change bf-projects directory to bf_projects?

Reply:
This dir should be not "python-importable".
So I'd keep "-" to prevent this.